### ISSUE TITLE
official Cascade workflow won't accept kohya-ss gui trained checkpoint (14gb stage c trained)

### ISSUE THREAD

**Issue by brentjohnston** [comfyanonymous/ComfyUI#2893]
Sorry in advance if this comes across as too wordy or technical here:

I am trying out the official Comfyui workflow, with a finetuned stage c and finetuned text encoder trained for Stable Cascade. The official workflow is slightly modified with a load clip node containing a trained text_model.safetensors (text encoder model) going into the positive and negative prompt. 

I am Testing training results with the kohya-ss gui dev via this post https://github.com/bmaltais/kohya_ss/issues/1982

I can only get the trained models to work in Comfyui with unet nodes at this time. 

The official workflow works for me with default stable_cascade_stage_c.safetensors when I test it, but I haven't specifically trained that newer 9gb checkpoint, only the 14gb one.

I am having this error when loading the trained stage c checkpoints with official workflow:

![5](https://github.com/comfyanonymous/ComfyUI/assets/23625562/87bacea3-6151-436a-bc75-eab9441ee88a)

![4](https://github.com/comfyanonymous/ComfyUI/assets/23625562/5b52b0b9-bce5-44ba-9d48-6e90b25d23fc)

Any ideas? Also do we have any info on why it was changed to not use the unet nodes anymore? I'm hoping this new workflow will resolve some issues with my finetunes, as they are not really listening to the prompts well. Unless I specify a lower trained text_model.safetensors (text encoder model) than a finetuned stage c model. 

That could be a separate issue though and maybe I need to increase the learning rate for stage c and decrease for text encoder training. More on that in the thread link.

**Comment by jac3km4** [comment]
You can convert the checkpoint into a ComfyUI compatible format with a workflow:

![image](https://github.com/comfyanonymous/ComfyUI/assets/11986158/5b13296a-f27a-45ef-896e-cb83a9536f5a)

**Comment by brentjohnston** [comment]
Thanks a lot I will give it a shot!
